main.go

```diff
@@ -145,3 +126,26 @@ func run(params commandParams) (*ldtest.Results, error) {
 
 	return &results, nil
 }
+
+func loadSuppressions(params *commandParams) error {
```

Comment:
What's the purpose of passing `params` as a pointer instead of a value? You're not modifying anything in it.

Reply:
Via `(t *TestIDPatternList).Set()` (line 143) it's modifying the `filters.MustNotMatch` member of `params`. That  does an append to the slice.

Reply:
🤦  Sorry, apparently I can't read